pnpm-lock.yaml

```diff
@@ -32372,6 +32422,87 @@ packages:
       - supports-color
     dev: true
 
+  /jest-config/29.7.0_@types+node@20.10.5:
```

Comment:
I don't think anything in the new package needs this... not sure how it made it into the updated lockfile. Maybe once again, revert lockfile to how it looks in main, and do a "clean" `pnpm i` to see if it takes only the necessary changes?